### ISSUE TITLE
Dataset used for training

### ISSUE THREAD

**Issue by xiayongk** [JaidedAI/EasyOCR#337]
Hello. I'm curious about the dataset you used for training.
I know you used synthetic data.
When creating a dictionary(especially ko.txt(korean dict)), I wonder if it was created in syllable units or included words or sentences.

Also, I fine-tuned your model using my synthetic dataset(word & sentence).
validation and test results were good(98%), but the result of using my fine-tuned model in EasyOCR was very terrible!
In conclusion, my fine-tuned model did not recognize a single letter. omg...:(
If you know the reason, please let me know.
I waiting for your answer.

**Comment by KaiserW** [comment]
Hello, I have also fine-tuned a model and got a new .pth file.
Could you please share some ideas about using this new .pth in EasyOCR?
Thanks a lot in advance!

**Comment by groupgithub21** [comment]
Hi @xiayongk @KaiserW ,
     How do you do finetuning in easyocr ? Is it by running the CRNN repo ? Would be grateful if you guys could elaborate ?
Btw, jaided.ai also has an option for finetuning (Though I am not sure what is being done) ?
http://www.jaided.ai/projects/finetune
![image](https://user-images.githubusercontent.com/79553071/122870121-d68f4700-d35f-11eb-8869-9a307a4511f5.png)

**Comment by rkcosmos** [comment]
Please see [this note](https://github.com/JaidedAI/EasyOCR/blob/master/custom_model.md).
